### PR TITLE
Homeworks 06

### PR DESCRIPTION
**Pull request by soosanin2**
homevork 07 created

### CODE REVIEW

File: homework_07/homework_07.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    # add: додати запис
+    # delete <name>: видалити запис за іменем (ключем)
+    # list: список всіх імен в книзі
+    # show <name>: детальна інформація по імені
+
+    # user_input = user_input.split()
+    # print(user_input)
+
+    try:
+        if user_input == "add":
```
Comment (klymenok) [inline homework_07/homework_07.py:20]: The task says that there shouldn't be overriding of records. Please, fix it

File: homework_07/homework_07.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            if user_input == "show":
+                show_input = input("enter a name: ")
+                str_num = str(phone_book.get(show_input))
+                print("Name " + show_input + " number " + str_num)
+
+            elif user_input == "delete":
+                del_input = input("enter a name: ")
+                # print((f'The subscribers "{del_input}" removed from the phone book'))
+                print("The subscribers " + del_input + " removed from the phone book")
+                del phone_book[del_input]
```
Comment (klymenok) [inline homework_07/homework_07.py]: It is better to check if key exists before deleting it

File: homework_07/homework_07.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                    all_sub = all_sub + ', ' + key_dct
+            print("All subscribers : " + all_sub)
+
+        if phone_book == {}:
+            print("Sorry, phone book is empty or you entered the wrong request")
+
+        else:
+            if user_input == "show":
+                show_input = input("enter a name: ")
+                str_num = str(phone_book.get(show_input))
```
Comment (klymenok) [inline homework_07/homework_07.py]: It is better to show some error is record does not exist

**Review by klymenok — CHANGES_REQUESTED** [review]

File: homework_07/homework_07.py (hunk 1 above)
Comment (soosanin2) [reply homework_07/homework_07.py:20]: But this is not a redefinition, but a comparison. I don't know what needs to be changed. 
just need to add a check for the presence of the key of the same name before the "add" command?

Comment (klymenok) [reply homework_07/homework_07.py:20]: The comment is about the whole of this part. Try to create a record `test` and then try to create it again. Users shouldn't be able to override records, but in your case, they can do it

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**